src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,49 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
+    private String password;
+    private byte[] salt;
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String login) {
+        this.email = login;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public byte[] getSalt() {
+        return salt;
+    }
+
+    public void setSalt(byte[] salt) {
+        this.salt = salt;
+    }
+
+    @Override
+    public String toString() {
+        return "{User} id = " + id
+                + ", email = " + email
+                + ", password = " + password + "}";
```

Comment:
Including the `password` in the `toString` method can be a security risk, as it might expose sensitive information in logs or other outputs. Consider removing the `password` from the `toString` method to prevent accidental exposure.